native_libs/src/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ target_compile_definitions(${PROJECT_NAME} PUBLIC _SILENCE_ALL_CXX17_DEPRECATION
 
 # Includes path: project root, arrow, third-party any-lite
 target_include_directories(${PROJECT_NAME} PUBLIC ${PROJECT_SOURCE_DIR} ${ARROW_INCLUDE} ${PROJECT_SOURCE_DIR}/../third-party/any-lite ${PROJECT_SOURCE_DIR}/../third-party/optional-lite ${PROJECT_SOURCE_DIR}/../third-party/variant ${RAPIDJSON_INCLUDE} ${DATE_INCLUDE} ${FMT_INCLUDE} ${PYTHON_INCLUDE_DIRS} ${PYTHON_NUMPY_INCLUDE_DIR} ${PYBIND_INCLUDE})
+target_link_libraries(${PROJECT_NAME} Boost::filesystem)
```

Comment:
You should be able to collapse this line and the one below into one

Reply:
I'd say that line is already too long and since we have "clean" way of expressing "adding" dependency I'll stay with split. But I'll group the commented line together with Boost include dirs, so they are grouped by dependency.